modules/installation-osp-kuryr-config-yaml.adoc

```diff
@@ -62,3 +62,8 @@ installer, so there is no need to set them. But if your environment does not
 meet both requirements, Kuryr SDN will not properly work. Trunks are needed
 to connect the pods to the {rh-openstack} network and Octavia is required to create the
 {product-title} services.
+
+[NOTE]
+====
+Be aware that CIDR ranges for networks are not adjustable after cluster installation. Red Hat does not provide direct guidance on determining the range because it requires careful consideration of the number of created pods per namespace.
```

Comment:
I am unsure about how knowing the following information is helping the user. Further, I am not sure what we imply by _direct guidance_ here. 
 _Red Hat does not provide direct guidance on determining the range because it requires careful consideration of the number of created pods per namespace._ 
 Would removing it be more helpful?

Reply:
Agree with Avani. 

Also, if this is very critical info to have, is it important enough to be a more severe admonition than a note? (I don't know, just thinking aloud)

Reply:
Thank you both so much!! 

@jeana-redhat That is a great point! I will change the severity to Important rather than just simply a note. 

@abhatt-rh I added the second half in case users assume we provide guidance on CIDR ranges during cluster installation. Would it be less confusing to the user to remove that sentence altogether?

Reply:
> Thank you both so much!!
> 
> @jeana-redhat That is a great point! I will change the severity to Important rather than just simply a note.
> 
> @abhatt-rh I added the second half in case users assume we provide guidance on CIDR ranges during cluster installation. Would it be less confusing to the user to remove that sentence altogether?

Thanks for sharing your thought behind adding it, @Wiharris. I leave it to your discretion :)

Reply:
> Agree with Avani.
> 
> Also, if this is very critical info to have, is it important enough to be a more severe admonition than a note? (I don't know, just thinking aloud)

Done! :)